LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/css/selectors/modal-pseudo-class-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+
+PASS Test that :modal matches modal <dialog>
+FAIL Test that :modal matches the fullscreen element assert_true: :fullscreen should match :modal expected true got false
```

Comment:
This failure is due to https://bugs.webkit.org/show_bug.cgi?id=248879 (orthogonal to this patch). Some other fullscreen tests were impacted as well.

A workaround would be waiting for the fullscreenchange event like the invalidation test does, though this is a test imported from upstream (unlike the invalidation test)